README.md

```diff
@@ -287,7 +287,7 @@ The jpo-geojsonconverter software system consists of the following modules hoste
 You may download the stable, default branch for ALL of these dependencies by using the following recursive git clone command:
 
 ```bash
-git clone --recurse-submodules https://github.com/usdot-jpo-ode/jpo-geojsonconverter.git
+git clone https://github.com/usdot-jpo-ode/jpo-geojsonconverter.git

```

Comment:
Still uses jpo-utils as a sub-module so --recurse-submodules is still needed.

Reply:
addressed